waspc/e2e-test/ShellCommands.hs

```diff
@@ -129,3 +135,17 @@ dockerBuild :: ShellCommandBuilder ShellCommand
 dockerBuild =
   return
     "[ -z \"$WASP_E2E_TESTS_SKIP_DOCKER\" ] && cd .wasp/build && docker build . && cd ../.. || true"
+
+copyGitTrackedFilesFromRepoPath :: Path' (Rel ProjectRoot) (Dir source) -> ShellCommandBuilder ShellCommand
+copyGitTrackedFilesFromRepoPath repoRelDirPath = do
+  context <- ask
+  let sourceDir = toFilePath projectRootFromGoldenTestProjectDir </> toFilePath repoRelDirPath
+      destinationDir = "./" ++ _ctxtCurrentProjectName context
+
+      createDestinationDir = "mkdir -p " ++ destinationDir
```

Comment:
What you can do is this
```suggestion
      createDestinationDir :: ShellCommand = "mkdir -p " ++ destinationDir
```
It makes it more obvious what this is, it would help me while reading this, that this is a shell command we are dealing with here. If this doesn't work out of the box, then there is extension from it, I am not sure now but I know it can be done.